.github/scripts/generate_ci_workflows.py

```diff
@@ -72,103 +63,127 @@ def generate_workflow_file(
 WINDOWS_WORKFLOWS = [
     PyTorchWindowsWorkflow(
         build_environment="pytorch-win-vs2019-cpu-py3",
+        test_runner_type="windows.4xlarge",
```

Comment:
It's probably still useful to have these as constants since they're not going to change much

Reply:
good point, done